### PR TITLE
WebDriver conformance release notes for Firefox 119

### PR DESCRIPTION
**Pull request by whimboo**
This PR is based on the [WebDriver conformance changes for Firefox 119](https://bugzilla.mozilla.org/buglist.cgi?component=Agent&component=Marionette&component=WebDriver%20BiDi&v1=fixed&columnlist=component%2Cresolution%2Cshort_desc%2Cbug_type%2Cstatus_whiteboard&resolution=FIXED&v2=%5Bwptsync%20downstream%5D&query_format=advanced&o1=equals&product=Remote%20Protocol&f2=status_whiteboard&o2=notsubstring&f1=cf_status_firefox119&list_id=16767067).

@lutien and @juliandescottes can you please review? Thanks

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/119/index.md [hunk 1]
```diff
@@ -71,10 +71,30 @@ This article provides information about the changes in Firefox 119 that affect d
 
 ### WebDriver conformance (WebDriver BiDi, Marionette)
 
+#### General
+
+- When performing a `pointerDown` action with the middle or right mouse button pressed, the `mousedown` event as emitted by the related HTML element had the value of the `buttons` property swapped ([Firefox bug 1850086](https://bugzil.la/1850086)).
+
+- When performing a `scroll` action of input type `wheel` with an origin set to `pointer` an `invalid argument` error was inappropriately raised, whereby as per the current WebDriver specification this combination is currently not supported ([Firefox bug 1850166](https://bugzil.la/1850166)).
```
Comment (juliandescottes) [inline files/en-us/mozilla/firefox/releases/119/index.md:78]:
```suggestion
- When performing a `scroll` action of input type `wheel` with an origin set to `pointer` an `invalid argument` error was inappropriately raised, whereas per the current WebDriver specification this combination is not supported ([Firefox bug 1850166](https://bugzil.la/1850166)).
```

Two nits I updated on the blog post: I don't think `whereby as` means what you expect here. And current/currently feels redundant, we should only keep one

**Review by juliandescottes — COMMENTED** [review]
Just one nit, otherwise lgtm. Thanks!

**Comment by lutien** [comment]
Apart from comments from Julian, I don't see any other issues, thanks!

File: files/en-us/mozilla/firefox/releases/119/index.md (hunk 1 above)
Comment (whimboo) [reply files/en-us/mozilla/firefox/releases/119/index.md:78]: These suggestions sounds fine to me. Thanks.

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
Thanks @whimboo

**Pull request merged**